docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/controller-install-was-proc.adoc

```diff
@@ -0,0 +1,51 @@
+[id='controller-install-was-proc']
+= Installing the Controller
+
+This section describes how to install the Controller on Oracle Weblogic Server.
+
+You can install the Controller and use the REST API or the {KIE_SERVER} Java Client API to interact with it.
+
+.Prerequisites
+* An IBM WebSphere Application Server instance configured as described in this document
+* Sufficient user permissions to complete the installation.
+* The `rhdm-7.0-controller-ee7.zip` file, extracted to a temporary directory. To download this file:
++
+--
+.. Click *Download* next to *{PRODUCT} 7.0.0 Add Ons* on the Product Downloads page of the https://access.redhat.com[Red Hat Customer Portal].
+.. Unzip the `rhdm-7.0.0.GA-add-ons.zip` file. The `rhdm-7.0-controller-ee7.zip` file is in the unzipped directory.
+--
```

Comment:
Users also need to repack Controller into war file. The steps are similar to ones described in Decision server chapter "4.2. Extracting Decision Server"